### DISCUSSION TITLE
src folder cnfigurable

### DISCUSSION THREAD

**Discussion by st3phani** [markshust/docker-magento#468] [Ideas]
@markshust can you create configurable name for `src` folder from `.env` file?

Thanks

**Comment by st3phani** [comment]
Why it helps to make the **src** folder configurable:
There is a possibility that the structure of the project in the repository will be different from just **Magento**.

**Comment by markshust** [comment]
Isn't the only place it is defined is in the `docker-compose.dev.yml` file? You can update it right there.
